.azure-pipelines.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Azure Pipelines configuration for building and testing react on Linux, Windows, and macOS.
+#
+
+trigger:
+- master
+
+jobs:
+- job: LinuxBasic
+  displayName: 'Linux: Lint and checks'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - template: scripts/azure-pipelines/linux-setup.yml
+    - template: scripts/azure-pipelines/linux-basic-steps.yml
+
+#- job: LinuxProd
+#  displayName: 'Linux: Prod and Fire'
+#  pool:
+#    vmImage: 'ubuntu-16.04'
+#  steps:
+#    - template: scripts/azure-pipelines/linux-setup.yml
+#    - template: scripts/azure-pipelines/linux-prod-steps.yml
+
+#- job: LinuxArtifacts
+#  displayName: 'Linux: Build and upload artifacts'
+#  pool:
+#    vmImage: 'ubuntu-16.04'
+#  steps:
+#    - template: scripts/azure-pipelines/linux-setup.yml
+#    - template: scripts/azure-pipelines/linux-artifacts-steps.yml
+
+#- job: LinuxCoverage
+#  displayName: 'Linux: Test coverage'
+#  pool:
+#    vmImage: 'ubuntu-16.04'
+#  steps:
+#    - template: scripts/azure-pipelines/linux-setup.yml
+#    - template: scripts/azure-pipelines/linux-coverage-steps.yml       
+
+- job: WindowsTests
+  displayName: 'Windows: Tests'
+  pool:
+    vmImage: 'vs2017-win2016'
+  steps:
+    - template: scripts/azure-pipelines/windows-setup.yml
+    - template: scripts/azure-pipelines/windows-tests.yml
+
+- job: WindowsLint
```

Comment:
Not sure it's necessary to run "Lint" on Windows (or mac). This job/task **should** be platform-agnostic, and any platform-specific error would be more of a tooling problem (rather than a code/change problem).